docs/README.md

```diff
@@ -175,7 +175,7 @@ with a rule like [`copy_to_bin`](https://docs.aspect.build/aspect-build/bazel-li
 ### Using binaries published to npm
 
 rules_js automatically mirrors the `bin` field from the `package.json` file of your npm dependencies
-to a Starlark API you can load from in your BUILD file or macro.
+to a Starlark API you can load from in your BUILD file or macro. Each mirrored `bin` entry is a [js_binary](./js_binary.md) supporting the same attributes such as `data`, `node_options` etc.
```

Comment:
not quite right. the `<bin>` is a `js_run_binary`, `<bin>_binary` a `js_binary` and `<bin>_test` a `js_test`

Reply:
you could put those in the table below instead of up here?